use/DifferentiationSNP.Rmd

```diff
@@ -62,23 +58,23 @@ dim(Mydata)
 ```
 
 ```{r, data_manipulate}
-ind <- as.character(Mydata$tree_id) #use later with adegent (individual labels)
-population <- as.character(Mydata$state) #use later with adegenet (population labels)
+ind <- as.character(Mydata$tree_id) # use later with adegenet (individual labels)
+population <- as.character(Mydata$state) # use later with adegenet (population labels)
 county <- Mydata$county 
-dim(Mydata) #550 individuals x 3082 SNPs
+dim(Mydata) # 550 individuals x 3082 SNPs
 ```
 
-Data conversion
+### Data conversion
 
-To convert MydataR to genind objects (adegent), the object locus needs to
-contain only genotypes. We decrease the number of snp to make the calculations
-faster and keep only 20 SNPs in the object locus. We convert Mydata1 in a fstat
+To convert Mydata to a genind object (adegenet), the input should only contain
```

Comment:
This makes sense. Perhaps I'll create an additional pull request for a style sheet in CONTRIBUTING

Sent from my iPhone

> On Dec 22, 2015, at 08:30, Hilmar Lapp <notifications@github.com> wrote:
> 
> In use/DifferentiationSNP.Rmd:
> 
> >  
> > -To convert MydataR to genind objects (adegent), the object locus needs to
> > -contain only genotypes. We decrease the number of snp to make the calculations
> > -faster and keep only 20 SNPs in the object locus. We convert Mydata1 in a fstat
> > +To convert Mydata to a genind object (adegenet), the input should only contain
> I wonder whether we should consider and settle on consistency for formatting of R object names. Personally, I would put everything that's an R object, perhaps with the exception or R package names, in backtics, so they are clearly distinguished from English language words, and as a signal to the reader that these will occur or have occurred in, and thus directly relate to, the preceding or following code chunks. Right now, sometimes data structure references are not backtic'ed, sometimes they are, and function names mostly seem to be. Perhaps it's not too early to start making these consistent, especially for a PR like this that's about "linting".
> 
> —
> Reply to this email directly or view it on GitHub.
>